### ISSUE TITLE
How to install for expo?

### ISSUE THREAD

**Issue by hristowwe** [berkelmas/rn-instagram-image#1]
Is it possible to install it for expo?

**Comment by berkelmas** [comment]
Yes the only dependency is react-native-gesture-handler and it is a built-in in Expo. So you should be able to npm install rn-instagram-image and use it 👍 @hristowwe
